cores/arduino/WInterrupts.cpp

```diff
@@ -19,14 +19,14 @@
 #include "WInterrupts.h"
 #include "Arduino.h"
 
-#ifdef __cplusplus
- extern "C" {
-#endif
+// #ifdef __cplusplus
+//  extern "C" {
+// #endif
```

Comment:
Rather than commenting out lines, please remove them - git is there to keep track of changes.
this applies to all the lines in this commit where // was used